assets/src/editor/events/dates-and-times/editor-date/grid-view/editor-date-grid-item.js

```diff
@@ -56,14 +60,29 @@ class EditorDateGridItem extends Component {
 				);
 			case 'start' :
 			default :
+				const start = eventDate.start.toFormat( 'h:mm a' );
 				return <BiggieCalendarDate
 					date={ eventDate.start }
 					htmlClass={ sidebarColorClass }
 					headerText={ __( 'starts', 'event_espresso' ) }
-					footerText={ eventDate.start.toFormat( 'h:mm a' ) }
+					footerText={ [ start, dateStatus ] }
 				/>;
 		}
 	};
+
+	/**
+	 * @function
+	 * @param {Object} event        model object defining the Event
+	 * @param {Object} eventDate    model object defining the Event Date
+	 * @return {Object} eventDate
+	 */
+	toggleLock = ( event, eventDate ) => {
+		if ( isModelEntityOfModel( eventDate, 'datetime' ) ) {
+			eventDate.isPrimary = ! eventDate.isPrimary;
+		}
+		return eventDate;
+	};
```

Comment:
it's not, I only used this field to test locking
sry I should have added some notes to the pull to explain some of this.
basically I think we need a way to prevent REM from modifying individual dates that have been customized.

example:

REM is used to generate an event that recurs once a week for 8 weeks, so 8 datetimes. After initial generation, those become actual datetimes and appear in the Event Date entity list, and can be edited just like any other Event Date. If for some reason the admin needs to change the details for ONE of those Event Dates, they can.

BUT...

if they THEN use REM to edit the recurrence pattern and/or change details for those Event Dates, the changes would be applied to **ALL** of the Event Dates in the series, which means that they would have to:

- remember which Event Dates had previously been customized
- re-apply those customizations every time a Recurrence pattern is edited

So because of this, I'm very likely going to apply the ability to lock entities. Just to flesh out how that would fit into the UI, I used the existing `isPrimary` field since it is not being used for anything else.

What I will most certainly do is add a new field to any model object that we want to lock. I'm thinking of using either a string or int for the lock field which would allow us to enable locking with greater control than just a boolean. For example, with Tickets we could have a basic edit lock that admins can turn on  or off, as well as a price lock that is applied to any Tickets with approved registrations which prevents the price from being modified (similar to what we have now). IF an admin needs to edit the price for a "price locked" ticket, they would have to duplicate that ticket and then archive the one that is "price locked" so that it can no longer be purchased.

My apologies for not explaining this stuff up front, I was rushing to get things done and failed to consider how confusing this would all look with no explanation.